torch/csrc/jit/script/strtod.cpp

```diff
@@ -29,6 +29,7 @@
 #define D_PINF ((double)+INFINITY)
 
 namespace {
+#ifdef __ANDROID__
```

Comment:
Just curios, how did you figure out that this code is only used for Android?

Reply:
if you search for the function name, it's only used in the android part below (and it's a local function)